custom/enikshay/management/commands/nikshay_registration_notification_time_report.py

```diff
@@ -76,17 +79,14 @@ def handle(self, days, *args, **options):
                 })
 
             if email_to:
-                attachment = {
-                    'title': result_file_name,
-                    'mimetype': 'text/csv',
-                    'file_obj': csvfile,
-                }
-
-                message = "Report for time taken for registration notifications yesterday till now"
-                send_html_email_async(
+                email_to = list(email_to) if not isinstance(email_to, basestring) else [email_to]
+                csvfile = open(result_file_name)
+                email = EmailMessage(
                     subject="Nikshay Registration Notification Time Report",
-                    recipient=email_to,
-                    html_content=message,
-                    text_content=message,
-                    file_attachments=[attachment]
+                    body="Report for time taken for registration notifications for %s day(s)" % self.days,
+                    to=email_to,
+                    from_email=settings.DEFAULT_FROM_EMAIL
                 )
+                email.attach(filename=result_file_name, content=csvfile.read())
```

Comment:
Does this need a mimetype? `text/csv`

Reply:
I think its determined automatically when not passed.

from [docs](https://docs.djangoproject.com/en/dev/topics/email/)
```
 mimetype is the optional MIME type for the attachment. If you omit mimetype, the MIME content type will be guessed from the filename of the attachment.
```